example/ffmpeg.js

```diff
@@ -1,32 +1,44 @@
 // Install fluent-ffmpeg before running this!
 const readline = require('readline');
-const path   = require('path');
-const fs     = require('fs');
-const ytdl   = require('..');
+const path = require('path');
+const fs = require('fs');
 const ffmpeg = require('fluent-ffmpeg');
+const ytdl = require('..');
 
-const url = 'https://www.youtube.com/watch?v=TGbwL8kSpEk';
```

Comment:
why change the video to download? it fits the use case more to use a music video, I'd imagine most people wanting to do this want to get the highest quality possible both in audio and video, and thus are more likely to do it for a medium where that's more important

plus, using a video that's a little bit long, but not too much, will help show the progress animated in the terminal. which users might find helpful.

Reply:
did you see this comment?